app/controllers/flights_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FlightsController < ApplicationController
+  def index
+    @flights = Flight.all
+  end
+
+  def destroy
+    flight = Flight.find(params[:flight_id])
+    passenger = Passenger.find(params[:id])
+    flight.passengers.delete(passenger)
+
+    redirect_to flights_path
+  end
```

Comment:
Talked about this in your routes file, but this is not the correct controller for this operation.